docs/excel/custom-functions-troubleshooting.md

```diff
@@ -38,6 +38,22 @@ Generally, these errors correspond to the errors you might already be familiar w
 
 Information about custom functions is cached by Office. Sometimes while developing and repeatedly reloading an add-in with custom functions your changes may not appear. You can fix this by clearing the Office cache. For more information, see [Clear the Office cache](../testing/clear-cache.md).
 
+### Clear the custom functions cache when your add-in runs
+
+There may be times when you need to clear the custom functions cache for an add-in deployed to your end users, so that add-in updates and custom functions setting changes are incorporated at the same time. Without triggering a custom functions cache clear, custom functions setting changes may take up to 24 hours to reach your end users.
```

Comment:
So I assume "add-in updates" are changes you made to code (task pane, custom functions). Are "custom functions setting changes" only changes to functions.json (basically function signature or attributes)?

Reply:
The cache clear updates functions.js and functions.json, I'll clarify this. Thanks!